### DISCUSSION TITLE
node-ux-latest constantly re-released

### DISCUSSION THREAD

**Discussion by eballeste** [bnpr/Malt#365] [Feedback & Feature Requests]
I keep getting release notifications related to this branch but not seeing any new commits, is there some type of GitHub action that is running on auto-pilot?

**Comment by pragma37** [comment]
Seems like the automatic tagging is not working on that branch for some reason.

But the releases are synced to the actual commits and the packed releases seem to be using the latest committed code.

We are working on improving the built-in nodes and, since we are two people working on the same code and there's a lot of design-related stuff to discuss, we are committing very often (#348).

You can disable the Release notifications for this repo:
![imagen](https://user-images.githubusercontent.com/36610999/181308699-c2cb9b34-8be7-4e6f-8244-708ea9415a10.png)
But afaik, there's no way to disable them only for pre-releases or for certain branches.

**Reply by eballeste** [reply]
sorry for this very unrelated comment but I have noticed that I don't originally understand things the first time I read them, I re-read your comment and somehow missed the fact that you are actually updating the branch and the releases are legit! 

Thanks for sharing the link to the discussion! I love reading though it even though I don't understand 95% of it.
